### DISCUSSION TITLE
PDE solution accuracy on differently shaped domains

### DISCUSSION THREAD

**Discussion by marijanarra** [lululxvi/deepxde#962] [Q&A]
Hi Lu Lu, 

I want to solve a Laplace equation on different polygonal domains with Neumann BCs dependent on lengths of the edges and surfaces of the corresponding polygons. 

I know that PINNs are scale dependent so I'm concerned about the difference in accuracy of the solutions I would be getting for differently shaped polygons. On the pics below I plotted the error distribution (colorcoded) for the each polygon for 20x2 NN and converged losses. I cannot make sense of the error distributions. Do you maybe have some thoughts on how to approach these kind of domain-scale sensitive problems?

Thanks in advance for your input!

![error_fixed](https://user-images.githubusercontent.com/64425956/195138913-c0e18661-ff46-4991-9652-2c329cdd6b8c.png)
![training_fixed_source_20_40x2_without_training_losses_200aeps_50meps](https://user-images.githubusercontent.com/64425956/195522263-bc959fff-b5de-459f-a213-061bbac2b8c3.png)

**Comment by lululxvi** [comment]
Unfortunately, I don't have experience with this.
